CIP-0114/README.md

```diff
@@ -0,0 +1,149 @@
+It should not be re-defined.
+
+For example `ED25519-BIP32` public keys are commonly encoded as a byte array.
+The array is 32 bytes long with the most significant byte of the key appearing first in the array.
+If a Tag is defined for this public key, it should simply define the Tag.
+Its Canonical encoding MUST follow this common encoding scheme.
+
+If a CIP does not refer to a Tag, nor the Canonical encoding specification for the data structure,
+AND it does not define an alternative encoding.
+Then the application implementing the encoding should assume it is encoded canonically.
+This helps ensure backward compatibility with pre-existing CIPs where Tags are not used.
+
+### Tag Registry
+
+Similar to [CIP-0010] this CIP defines a registry of all known tags.
+The format of the registry is defined by the json schema: [CIP Tag Registry Schema].
+New entries MUST be added to the [CIP Tag Registry] in a PR for a CIP that first defines a new CBOR Tag.
+They MUST be updated when the Tag is accepted or rejected by IANA.
+The registry clearly notes if the tag is currently known to be registered or not.
+If a Tag is not yet registered then any implementor must be aware that its possible the Tag number could change and is not final.
+Unregistered tags MUST not be used in any main net on-chain metadata or data structures.
+They should only be used for testing purposes until registration is complete.
+
+## Rationale: how does this CIP achieve its goals?
+
+Creating a [registry][CIP Tag Registry] for `CIP Tag` values has the following benefit:
+
+1) It makes it easy for developers to know which `CIP Tags` to use, and if they have been registered or not.
+2) It makes it easy to avoid collisions with other standards that use CIP Tags.
+3) It helps CIP authors to find appropriate CIP tags for their use case, or to define new tags.
+
+The process for defining and registering Tags should help provide clarity about how a CIP tag can be defined.
+It also provides clarity on the responsibility CIP authors have to register the tags they create.
+If a CIP author is not prepared to take on that responsibility they should not create a Tag.
+
+## Path to Active
+
+### Acceptance Criteria
+
+- [ ] [CPS-0014] is accepted.
+- [ ] At least 1 CIPs are accepted into the Tag registry for historical tags.
+- [ ] At least 1 CIPs are accepted into the Tag registry for new tags.
+- [ ] At least 3 CIPs of any kind are accepted into the Tag registry.
+
+### Implementation Plan
+
+- [ ] Author to write the first CBOR tag CIP.
+
+## References
+
+- [CPS-0014 -  Register of CBOR Tags for Cardano Data structures][CPS-0014]
+- [RFC8949 - Concise Binary Object Representation (CBOR)][CBOR]
+- [IANA CBOR Tag Registry][IANA]
+
+## Copyright
+
+This CIP is licensed under [CC-BY-4.0](https://creativecommons.org/licenses/by/4.0/legalcode)
+
+[CPS-0014]: https://github.com/cardano-foundation/CIPs/pull/751
```

Comment:
```suggestion
[CPS-0014]: https://github.com/cardano-foundation/CIPs/tree/master/CPS-0014
```
preempt the merge

Reply:
@Ryun1 yes; @stevenj yes please commit this now rather than updating later... otherwise we might miss our chance to correct it, especially if this gets merged between meetings as we are talking about doing.

Reply:
Done